voluncheer/requirements.txt

```diff
@@ -17,6 +17,7 @@ django-widget-tweaks==1.4.12
 django_debug_toolbar==3.8.1
 flake8==5.0.4
 jmespath==1.0.1
+mock==5.0.1
```

Comment:
I don't think mock is used and thus should be removed. Plus, mock is now built into the `unittest` package: https://docs.python.org/3/library/unittest.mock.html